README.md

```diff
@@ -1,2 +1,35 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+Flowchart : https://i.gyazo.com/4f236f8ca944556c4a966e149faec5b7.png
+
+points = 0
+secretWord = "Yeet"
+firstLetter = "Y"
+secondLetter = "E"
+thirdLetter = "E"
+fourthLetter = "T"
+print ("What is the first letter?")
+firstLetterGuess = input()
+print ("What is the second letter?")
+secondLetterGuess = input ()
+print("What is the third letter?")
+thirdLetterGuess = input ()
+print ("What is the fourth letter?")
+fourthLetterGuess = input ()
+print("Your guess was: " + firstLetterGuess + secondLetterGuess + thirdLetterGuess + fourthLetterGuess)
+
+if firstLetter equals firstLetterGuess
+  points += 1
+if secondLetter equals secondLetterGuess
+  points += 1
+if thirdLetter equals thirdLetterGuess
+  points += 1
+if fourthLetter equals fourthLetterGuess
```

Comment:
This approach is also hard-coded to the length of the word being 4. Using a loop would make the pseudocode more general and would also cut down on repetition. For example:
```
for i = 0, ..., wordLength - 1
   if secretWord[i] == guess[i]
      points += 1
```